cmake/cpackEngineRPM.cmake

```diff
@@ -71,20 +71,14 @@ IF (EXISTS "/etc/redhat-release")
     string(REGEX MATCH "release ([0-9]+)" CENTOS "${REDHAT_VERSION}")
     set(REDHAT_VERSION_NUMBER "${CMAKE_MATCH_1}")
 ENDIF ()
+
 IF (EXISTS "/etc/SuSE-release")
     file (READ "/etc/SuSE-release" SUSE_VERSION)
     string(REGEX MATCH "VERSION = ([0-9]+)" SUSE "${SUSE_VERSION}")
     set(SUSE_VERSION_NUMBER "${CMAKE_MATCH_1}")
 ENDIF ()
-if (${REDHAT_VERSION_NUMBER} EQUAL 6)
-    SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "MariaDB-columnstore-shared" "snappy" "net-tools" "MariaDB-server" "python3")
-    # Disable auto require as this will also try to pull Boost via RPM
-    SET(CPACK_RPM_PACKAGE_AUTOREQPROV " no")
-elseif (${SUSE_VERSION_NUMBER} EQUAL 12)
-    SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "boost-devel >= 1.54.0" "libsnappy1" "net-tools" "MariaDB-server" "python3" "jemalloc")
-else ()
-    SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "boost >= 1.53.0" "snappy" "net-tools" "MariaDB-server" "python3" "jemalloc")
-endif()
+
+SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "snappy" "net-tools" "MariaDB-server" "python3" "jemalloc" "procps-ng")
```

Comment:
elfutils for eu-stack?

Reply:
There is the logic difference, procs is required for Columnstore to install, we use ps in install scripts, eustack is used on ci only, and IMHO should be installed separately